src/components/EditCoursePage/index.jsx

```diff
@@ -45,11 +46,29 @@ class EditCoursePage extends React.Component {
           key,
           uuid,
           entitlements,
+          course_runs: initialCourseRuns,
         },
       },
       editCourse,
     } = this.props;
     const courseData = options;
+    const modifiedCourseRuns = [];
+    const newCourseRuns = [];
+    courseData.course_runs.forEach((modifiedCourseRun) => {
+      let found = false;
+      initialCourseRuns.forEach((initialCourseRun) => {
+        if (initialCourseRun.key === modifiedCourseRun.key) {
+          found = true;
```

Comment:
The `found` makes me think that `.some()` would be useful here, but maybe it doesn't work cleanly with pusing multiple runs